### ISSUE TITLE
[Bug] 'Normal' game speed looks choppy

### ISSUE THREAD

**Issue by masterofobzene** [CorsixTH/CorsixTH#2679]
### Describe the issue

'Normal' game speed looks choppy and 'Maximum speed' looks like actual 'Normal' speed.

Windows 10 x64. 
GoG game files.
Nvidia 3080 gpu
AMD x5900 cpu
16 gb ram.

### Steps to Reproduce

1.  Compare between those speeds.

https://github.com/user-attachments/assets/9347d5a7-eb6a-47b7-99df-d8ad31f7570a

### Expected Behaviour

Normal speed should at least look like "maximum speed" and I suppose "maximum speed" should be faster.

### Have you enabled any current CorsixTH Experimental Features?

- [ ] Right mouse panning
- [ ] Falling actions

### Save Game

Save game attached to report: Yes/No

### CorsixTH Version

0.68

### Operating System

Windows 10 x64

### Theme Hospital Version

GOG.com

### Gamelog.txt

```yaml
Gamelog generated on 2023-11-11 00:59:47
Compiled with api_version: 2684, arch: x64, audio: true, os: windows
SDL renderer: direct3d
Lua 5.4 run with api version: 2684, game version: v0.67, savegame version: 180
```

### Additional Information

This is all it has inside the gamelog.txt, and seems old due to the date it has.
No tampering done and it's the only gamelog.txt that exists.

**Comment by MandrakeHorse** [comment]
0.68.0 included a change where "Game speeds are now more closely aligned with the original game", which was implemented by #1494. Normal speed was changed from `{1, 3}` to `{2, 3}` (2 ticks per 108ms = 1 tick per 54 ms) but perhaps we should consider increasing it slightly to improve the framerate.

As a sidenote, there appears to be a bug with the DOS/GOG version of Theme Hospital where "Normal" speed is exactly the same as "Max Speed", so we should ideally be comparing CorsixTH's performance against the original CD/Windows version: https://www.gog.com/forum/theme_hospital/normal_speed_serious_game_bug

**Comment by ARGAMX** [comment]
I think this issue is about that game looks choppy, not really about game speed.
I have noticed that problem too, but I didn’t have an understanding of how to formulate it.

Original game not looks choppy. 
Corsix 68 looks choppy on normal speed (3). 
And sometimes have feeling that it choppy on the "And then some more" speed (5). But not sure.

I have checked link that @MandrakeHorse provided to 1494 task and found that speed 3 and 5 make 2 tick, and all other speeds make only 1 tick.

So it looks like making 2 tick per 'frame' made game looks choppy.
Especially it looks sad on normal speed (3).

@mugmuggy have you noticed that too?
